app/shared/hooks/use-search/useFetchStaticFilters.ts

```diff
@@ -12,8 +12,8 @@ export function useFetchStaticFilters<T = unknown>(endpoint: string | null) {
     queryKey: ['static-filters', endpoint ?? 'none', locale],
     queryFn: async (): Promise<T | null> => {
       if (!endpoint) return null;
-      const resp = await tableClientService.getTableStaticData<any>(endpoint, locale);
-      return resp as T;
+      const resp = await tableClientService.getTableStaticData<T>(endpoint, locale);
```

Comment:
Here you can simplify:
```suggestion
      return await tableClientService.getTableStaticData<T>(endpoint, locale);
```

Reply:
Thank you for review. Removed const in latest commit.